portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,347 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+/* For all pages */ 
+
+html {
+  height: 100%;
+  margin: 0;  
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'Work Sans';
+  height: 100%;
+  margin: 0;
+}
+
+h1 {
+  font-size: 60px;
+  font-weight: bold;
+}
+
+h2 {
+  font-size: 30px;
+}
+
+p {
+  font-size: 20px;
+}
+
+img {
+  max-height: 100%;
+  max-width: 100%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.topnav {
+  background: lavender;
+  overflow: hidden;
+  padding: 16px;
+}
+
+.topnav a {
+  color: black;
+  display: block;
+  float: right;
+  font-size: 20px;
+  padding: 14px 16px;
+  text-align: center;
+  text-decoration: none;
+}
+
+.topnav-left a {
+  color: black;
+  display: block;
+  float: left;
+  font-size: 20px;
+  padding: 14px 16px;
+  text-align: center;
+  text-decoration: none;
+}
+
+.topnav a:hover {
```

Comment:
Css guide to separate out declaration
https://engdoc.corp.google.com/eng/doc/devguide/htmlcss/styleguide.md?cl=head#Selector_and_Declaration_Separation

Reply:
Sorry the link doesn't work for me yet, but are you referring to this section: https://screenshot.googleplex.com/e8WEqZNKurT ?
 
I believe the ".topnav a" is actually 1 declaration because I was trying to specify it for all "a" tags inside ".topnav".